### DISCUSSION TITLE
How to center the banner subtitle?

### DISCUSSION THREAD

**Discussion by lukans** [templaza/astroid-framework#803] [Q&A]
How to center the banner subtitle? I added the CSS custom code but it doesn't work.
.astroid-banner .astroid-banner-inner .astroid-banner-subtitle {
  font-size: 20px;
  text-align: center;
}

Banner title work ok.
.astroid-banner .astroid-banner-inner .astroid-banner-title {
  font-size: 40px;
  text-align: center;
}

Live page is https://artmagazin.lwdesign.rs/index.php/pretraga

**Comment by sonvnn** [comment]
Because sub-title is a span tag. You have to add display:block;
```
.astroid-banner .astroid-banner-inner .astroid-banner-subtitle {
font-size: 20px;
text-align: center;
display:block;
}
```

It will work

**Comment by lukans** [comment]
It work. Many thanks.
